src/components/Form.js

```diff
@@ -40,6 +40,8 @@ const propTypes = {
     // eslint-disable-next-line react/forbid-prop-types
     draftValues: PropTypes.object,
 
+    enabledWhenOffline: PropTypes.bool,
```

Comment:
Add the same comment about this prop that we have at `FormAlertWithSubmitButton`